style.css

```diff
@@ -0,0 +1,93 @@
+body {
+background: url("http://wp.givingtuesday.ca/app/uploads/2014/11/GivingTuesday_BlogHeader.jpg") no-repeat center center fixed;
```

Comment:
Format this file. It still uses the http url instead the image you added. Please check your changes thoroughly before committing.